src/client/gui/pubspec.yaml

```diff
@@ -52,7 +52,10 @@ dependencies:
       url: https://github.com/google/flutter-desktop-embedding.git
       path: plugins/window_size
       ref: 6c66ad2
-  xterm: ^4.0.0
+  xterm:
+    git:
+      url: https://github.com/levkropp/xterm.dart
```

Comment:
This should live in the Canonical org, but I see there are similar cases here. I'll propose fixing all that on an upcoming pulse.